JPS_Ontology/ontology/ontodispersion/OntoDispersion.csv

```diff
@@ -0,0 +1,46 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
+OntoDispersion,TBox,https://www.theworldavatar.com/kg/ontodispersion/,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoDispersion,TBox,1.1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoDispersion,TBox,,http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+OntoDispersion,TBox,,http://www.w3.org/2002/07/owl#imports,,,,,,
+DispersionMatrix,Class,,,,,,URL to download the raw output file stored as time series,https://www.theworldavatar.com/kg/ontodispersion/,
+DispersionLayer,Class,,,,,,Name of the dispersion matrix layer in GeoServer stored as time series,https://www.theworldavatar.com/kg/ontodispersion/,
+ShipsLayer,Class,,,,,,Name of the ships layer in GeoServer stored as time series,https://www.theworldavatar.com/kg/ontodispersion/,
+Scope,Class,http://www.opengis.net/ont/geosparql#Feature,IS-A,,,,Geospatial extend defining the simulation box as rectangle,https://www.theworldavatar.com/kg/ontodispersion/,
+PointSource,Class,,,,,,A point source emitting emissions,https://www.theworldavatar.com/kg/ontodispersion/,
+DynamicPointSource,Class,PointSource,IS-A,,,,A moving point source,https://www.theworldavatar.com/kg/ontodispersion/,
+StaticPointSource,Class,PointSource,IS-A,,,,A static point source,https://www.theworldavatar.com/kg/ontodispersion/,
+Ship,Class,DynamicPointSource,IS-A,,,,,https://www.theworldavatar.com/kg/ontodispersion/,
+Emission,Class,,,,,,Superclass of emission,https://www.theworldavatar.com/kg/ontodispersion/,
+nx,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,Number of cells in x direction,https://www.theworldavatar.com/kg/ontodispersion/,
+ny,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,Number of cells in y direction,https://www.theworldavatar.com/kg/ontodispersion/,
+z,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Height,IS-A,,,,Height to simulate,https://www.theworldavatar.com/kg/ontodispersion/,
+SimulationTime,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,Unix timestamp to run the simulation,https://www.theworldavatar.com/kg/ontodispersion/,
```

Comment:
Strictly speaking this is misleading, only the actual value will be unix timestamp

Reply:
May address this after tidying up the aermod agent